README.md

```diff
@@ -12,9 +12,9 @@ You can find the deployed project at [🚫URL NAME GOES HERE](🚫copy and paste
 
 Add contributor info below, make sure add images and edit the social links for each member. Add to or delete these place-holders as needed
 
-| Student 1         | Aric Repp                                                                                                                  | Tommy Conner                                                                                                                   | Ryan Schueck                                                                                                                  | Student 5         | Student 6         | Germaine Pannell        | Reed Turgeon      |
-| :-:               | :-:                                                                                                                        | :-:                                                                                                                            | :-:                                                                                                                           | :-:               | :-:               | :-:               | :-:               |
-| student1-github   | [<img src="https://github.com/favicon.ico" width="25"> ](https://github.com/aricrepp)                                      | [<img src="https://github.com/favicon.ico" width="25"> ](https://github.com/tommyconner96)                                     | [<img src="https://github.com/favicon.ico" width="25"> ](https://github.com/Ryan-webdev)                                      | student5-github   | student6-github   | [<img src="https://github.com/favicon.ico" width="25"> ](https://github.com/germainep)   | [<img src="https://github.com/favicon.ico" width="25"> ](https://github.com/MrT3313)                          |
+|     Student 1     |                                                         Aric Repp                                                          |                                                          Tommy Conner                                                          |                                                         Ryan Schueck                                                          |     Student 5     |     Student 6     |                                                      Germaine Pannell                                                       |                                                         Reed Turgeon                                                         |
```

Comment:
oh look at that! it removed all the cleaned our spacing and added back that line! 

not an issue just remember to turn your text wrap off